content/millennium/r4/medications.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Medications | R4 API
+layout: api
+---
+
+<%= render '/medications.*' %>
```

Comment:
this is going to be a problem. this pulls in content from layouts/medications.html. all the other layouts are resource- and fhir-version-agnostic, but that one references specific resources and links to specific versions of hl7.org

we'll need to possibly split it into medications_r4.html and update its content appropriately

Reply:
Changed it to render new layout https://github.com/cerner/fhir.cerner.com/pull/255/commits/7f7bd5fd0fd7d7d18677ca1d5618107ccb707330#diff-5981708289046f1d63210fd0fbff1a7eR6

For the new layout, I used the same content but changed the link to point to r4. https://github.com/cerner/fhir.cerner.com/pull/255/commits/7f7bd5fd0fd7d7d18677ca1d5618107ccb707330#diff-d38f08e30b722da7ec389eb023068447R1

Reply:
need a trailing newline on that new template file

this url http://hl7.org/fhir/r4/resourceguide.html#3.1.2.3 doesn't lead anywhere (there's no 3.1.2.3 anchor on the page) - the r4 resource guide page is less comprehensive than the dstu2 one. I don't know if/where the r4 version of that content is

we also can't include the medorder vs. medstatement content - we don't expose either of those resources for r4 yet

Reply:
Discussed offline. Updated the new file.
https://github.com/cerner/fhir.cerner.com/pull/255/commits/b519a215146c45c38295cd8fae4b6bbf6b7363af#diff-d38f08e30b722da7ec389eb023068447L3